src/Components/Assets/AssetsList.tsx

```diff
@@ -228,6 +228,40 @@ const AssetsList = () => {
     }
   };
 
+  const assetIconGenerator = (name: any) => {
+    if (name === "ONVIF") {
+      return (
+        <svg
```

Comment:
Can we add svg in a different file and import it directly from there because currently 
1. We cannot reuse the same svg somewhere else if wish to in future.
2. SVG values in jsx would be tough to maintain